### DISCUSSION TITLE
Lesson 7 [FAIL. Reason: EvmError: Revert] YouTube: 53:31 / 10:40:14

### DISCUSSION THREAD

**Discussion by MIKEX818** [Cyfrin/foundry-full-course-cu#1850] [General]
Hello everyone, I have an error and could not figure it myself. Can someone help to look at and discuss the issue? Here is the context:

Terminal: 
<img width="1405" alt="Screenshot 2024-06-17 at 2 27 09 PM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/169244435/a2887b2d-dd5e-4f90-9c71-99367c251077">

FundMeTest.sol
<img width="724" alt="Screenshot 2024-06-17 at 2 28 02 PM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/169244435/ed648761-4719-4f61-9509-6c01d88f4108">

FundMe.sol
<img width="934" alt="Screenshot 2024-06-17 at 2 28 49 PM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/169244435/2d1c687a-f226-487d-836f-fec2c37b9558">

DeployFundMe.sol
<img width="585" alt="Screenshot 2024-06-17 at 2 29 29 PM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/169244435/0316babd-c8ef-48e3-b672-f3aed8a109b3">

HelperConfig.s.sol
<img width="928" alt="Screenshot 2024-06-17 at 2 30 01 PM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/169244435/c2c35a91-f74a-4ba8-a12e-d2939d0db133">

PriceConverter.sol
<img width="890" alt="Screenshot 2024-06-17 at 2 30 39 PM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/169244435/e8863821-9fa1-4ac6-ab01-06e8854ec4fb">

**Comment by EngrPips** [comment]
Hello @kaijieshao, Are you running this test on the Anvil network or the sepolia network? If you are running on the Anvil network, please show your `getAnvilEthConfig` function in your `HelperConfig.`
